Backend/dev/migrations/dynamic-offer-driver-app/0301-rental-feat.sql

```diff
@@ -80,10 +80,77 @@ ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN odometer_start_reading_image_
 ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN odometer_end_reading_image_id CHARACTER VARYING(255);
 
 ALTER TABLE atlas_driver_offer_bpp.driver_pool_config ADD COLUMN single_batch_process_time_rental bigint DEFAULT 60;
-ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN allocate_rental_ride_time_diff bigint DEFAULT 900;
+ALTER TABLE atlas_driver_offer_bpp.transporter_config ADD COLUMN allocate_rental_ride_time_diff bigint DEFAULT 1800;
 
 ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD COLUMN search_request_tag character varying(36) default 'ON_DEMAND';
 ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD COLUMN booking_id character(36);
 
 ALTER TABLE atlas_driver_offer_bpp.search_try ADD COLUMN tag character varying(36) default 'ON_DEMAND';
 ALTER TABLE atlas_driver_offer_bpp.search_try ALTER COLUMN estimate_id DROP NOT NULL;
+
+ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN next_stop_loc_id character(36);
+
+ALTER TABLE atlas_driver_offer_bpp.driver_pool_config ADD COLUMN search_request_tag character varying(36) NOT NULL default 'ON_DEMAND';
```

Comment:
check if it would cause any issues as it is NOT NULL col if it is added to KV or not